styles/main.css

```diff
@@ -14,7 +14,8 @@ nav a:link {color:#45A29E;
 }
 nav a:visited {color:#4F8775;
 }
-.byline {color:#D1E8E2; opacity: 0.5;
+.byline {
+    color: rgba(209, 232, 226, 0.5); 

```

Comment:
What you have works (no marks lost), but I was going for using the rgba syntax.  Also note: once we get to multiple declarations in a single CSS rule, please leave the opening { as the end of its line as I have done for the .byline rule